src/tini.c

```diff
@@ -164,6 +165,13 @@ void print_license(FILE* const file) {
 int parse_args(const int argc, char* const argv[], char* (**child_args_ptr_ptr)[], int* const parse_fail_exitcode_ptr) {
 	char* name = argv[0];
 
+	// We handle --version if it's the *only* argument provided.
+	if (argc == 2 && strcmp("--version", argv[1]) == 0) {
+		*parse_fail_exitcode_ptr = 0;
+		fprintf(stdout, "%s\n", TINI_VERSION_STRING);
+		return 1;
```

Comment:
(I should probably rename `parse_fail_exitcode_ptr ` to `parse_exitcode_ptr` to be a little clearer that this doesn't really indicate a "failure")